lib/recent.go

```diff
@@ -0,0 +1,107 @@
+package lib
+
+import (
+	"encoding/json"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+type RecentFiles struct {
+	Terraform []string `json:"terraform"`
+	OpenTofu  []string `json:"openTofu"`
```

Comment:
It's probably worth of lowercasing `openTofu` for consistency with `terraform`

Reply:
Will do